gapir/cc/interpreter.cpp

```diff
@@ -166,6 +171,9 @@ Interpreter::Result Interpreter::call(uint32_t opcode) {
   auto api = (opcode & API_INDEX_MASK) >> API_BIT_SHIFT;
   auto func = mBuiltins[api].lookup(id);
   auto label = getLabel();
+  if ((mNumApiCallsMade++ % LOG_EVERY_N_API_CALLS) == 0) {
```

Comment:
I would prefer if we sent the status every X% rather than N api calls. That would mean more reliable data coming out. Also that should PROBABLY be a function of the ReplayRequest rather than a hard-coded value.